benchmarks/run.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+const { writeFileSync } = require('fs')
+const { spawn } = require('child_process')
+const { resolve, join } = require('path')
+const autocannon = require('autocannon')
+const testCases = require('./testCases')
+const printReport = require('./printReport')
+
+const serverPath = resolve(join(__dirname, 'server.js'))
+
+const pauseFor = (ms) => new Promise((resolve) => {
+  setTimeout(resolve, ms)
+})
+
+let currentServerInstance = null
+const startServerFor = (test, port) => {
+  const env = Object.assign({}, test.options, { PORT: port, PATH: process.env.PATH })
+  currentServerInstance = spawn(
+    serverPath,
+    [],
+    {
+      env,
+      stdio: [ 'ignore', 'ignore', 'ignore' ]
```

Comment:
I think it is my problem "but" 😆 I need to add `shell: true` to run the bench and in any case my output is useless

```
Server exited with exit code 0
� No etag (2 Mb, buffer) �
```

need to fix my pc first, to run bench 😅

Reply:
Are you running this under windows?

Reply:
Yes

Reply:
I'll add the `shell` option as default then to make sure it works also under Windows. You have to add proper emojis support to your system though :D